src/app/organization/loan-provisioning-criteria/view-loan-provisioning-criteria/view-loan-provisioning-criteria.component.html

```diff
@@ -0,0 +1,80 @@
+<div class="container">
+
+  <mat-card>
+
+      <div fxLayout="row" fxLayoutAlign="start">
+        <div>
+          <h2>{{provisioningData.criteriaName}}</h2>
```

Comment:
@BLasan can you try getting title aligned with the buttons, i'll review #625 till then

Reply:
Ok

Reply:
Added inline style margin-top:-10px for the title. Is it ok?

Reply:
use a class instead
Also I am reviewing edit charge but since it's big and I have a call right now i'll get back to you in 3 hours
This looks good to merge however I will need to ask Abhay for design finalization.

Reply:
You mean bootstrap classes? Like p-t,m-t etc.. ? In-order to align the buttons and the title we have to reduce the margin top.

Reply:
no define a class in scss file

Reply:
Resolved this. Please take a look